lib/shopify_dashboard_plus.rb

```diff
@@ -90,11 +122,7 @@ def validate_date_range(from, to)
     interval_end = DateTime.parse(to) rescue nil
 
     if interval_start && interval_end
-      if interval_start <= interval_end
-        return true
-      else
-        return false
-      end
+      return true if interval_start <= interval_end
     end
```

Comment:
Given that the range is invalid if either one fails to parse, this could be simplified to

```ruby
def validate_date_range(from, to)
  DateTime.parse(from) <= DateTime.parse(to)
rescue ArgumentError
  false
end
```

Also, the convention for methods that return booleans is to end them with a `?` (ie. `date_range_valid?`).